compiler/compiler.ts

```diff
@@ -2177,19 +2177,17 @@ export class Compiler {
             const id_buffer = typeof id == "string" ? hex2buffer(id, Pointer.MAX_POINTER_ID_SIZE, true) : id;
             const pointer_origin = (id_buffer[0]==BinaryCode.ENDPOINT || id_buffer[0]==BinaryCode.PERSON_ALIAS || id_buffer[0]==BinaryCode.INSTITUTION_ALIAS) ? <IdEndpoint> Target.get(id_buffer.slice(1,19), id_buffer.slice(19,21), id_buffer[0]) : null;
             
-            // TODO: enable
-            // const singleReceiver = 
-            //     (SCOPE.options.to instanceof Endpoint && SCOPE.options.to) ||
-            //     (SCOPE.options.to instanceof Disjunction && SCOPE.options.to.size == 1 && [...SCOPE.options.to][0] instanceof Endpoint && [...SCOPE.options.to][0])
+            const singleReceiver = 
+                (SCOPE.options.to instanceof Endpoint && SCOPE.options.to) ||
+                (SCOPE.options.to instanceof Disjunction && SCOPE.options.to.size == 1 && [...SCOPE.options.to][0] instanceof Endpoint && [...SCOPE.options.to][0])
 
             if (
                 pointer_origin && 
                 SCOPE.options.preemptive_pointer_init !== false && // preemptive_pointer_init enabled
                 action_type == ACTION_TYPE.GET && // is get
                 Runtime.endpoint.equals(pointer_origin) &&  // is own pointer
-                SCOPE.options.to != Runtime.endpoint // not sending to self
-                // && TODO: enable - currently not useful because this prevents preemptive loading from working after frontend reload, because the endpoint is still considered a subscriber
-                // !Pointer.get(id)?.subscribers?.has(singleReceiver) // receiver is subscribed to pointer - assume it already has the current pointer value
+                SCOPE.options.to != Runtime.endpoint && // not sending to self
```

Comment:
What should happen with instances here? Is `SCOPE.options.to != Runtime.endpoint` enought?

Reply:
Preemptive pointers should still be sent to other instances - it just does not make sense to send them to the own endpoint instance that already has the pointer value.

Reply:
The edge case might be when you send it as `@@123/0001` to `@@123/0001` - you can't be certain to which endpoint the network routes the message - it might be your own endpoint, in which case the preemptive pointer initialization was pointless, but it might also be a remote endpoint. You can't really know at this point, so just always send it.